shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/java/org/apache/shardingsphere/sql/parser/oracle/visitor/statement/impl/OracleDDLStatementSQLVisitor.java

```diff
@@ -901,9 +903,14 @@ public ASTNode visitAlterMaterializedZonemap(final AlterMaterializedZonemapConte
     public ASTNode visitAlterJava(final AlterJavaContext ctx) {
         return new OracleAlterJavaStatement();
     }
-    
```

Comment:
Please execute `mvn spotless:apply` on your local machine to format code.

Reply:
Ok, I will correct the code format.

Reply:
Ok, I will format the code.